cloud/pkg/router/utils/path.go

```diff
@@ -0,0 +1,69 @@
+package utils
+
+import (
+	"regexp"
+	"strings"
+
+	"k8s.io/klog/v2"
+)
+
+const (
+	ParamRegex = "{[-A-Za-z0-9+&@#%?=~_|!:,.;]+}"
+	pathRegex  = "[-A-Za-z0-9+&@#%?=~_|!:,.;]+"
```

Comment:
`ParamRegex` is MixedCaps, `pathRegex`  is mixedCaps, what is the difference? 
I think use mixedCaps here.

Reply:
just reuse `pathRegex`: `paramRegex  = "{" + pathRegex + "}"`

Reply:
done